src/styles/blocks/default-styles.scss

```diff
@@ -0,0 +1,14 @@
+* {
+  // margin: 0;
+  // padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, Halvetica, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
```

Comment:
it's only allowed to use box-sizing : border-box at * selector. So move other rules to html/body tag selectors